include/mbedtls/ecdsa.h

```diff
@@ -171,31 +208,44 @@ int mbedtls_ecdsa_write_signature( mbedtls_ecdsa_context *ctx, mbedtls_md_type_t
 #define MBEDTLS_DEPRECATED
 #endif
 /**
- * \brief           Compute ECDSA signature and write it to buffer,
- *                  serialized as defined in RFC 4492 page 20.
- *                  Deterministic version, RFC 6979.
- *                  (Not thread-safe to use same context in multiple threads)
+ * \brief   This function computes an ECDSA signature and writes it to a buffer,
+ *          serialized as defined in <em>RFC-4492: Elliptic Curve Cryptography 
+ *          (ECC) Cipher Suites for Transport Layer Security (TLS)</em>.
+ *
+ *          The deterministic version, defined in <em>RFC-6979: 
+ *          Deterministic Usage of the Digital Signature Algorithm (DSA) and 
+ *          Elliptic Curve Digital Signature Algorithm (ECDSA)</em> is used
+ *          if #MBEDTLS_ECDSA_DETERMINISTIC is defined.
```

Comment:
Blocker: This function is always deterministic (in contrast to its superseding function `mbedtls_ecdsa_write_signature`, which is a bit confusing).